### PR TITLE
AddressTypeChange for CE Case ceActualResponses Defaulted to Zero

### PR DESCRIPTION
**Pull request by LeoKHoward**
# Motivation and Context
When an AddressTypeChange event occurs where the new address is a CE Case, the ceActualResponses is null rather than 0 

# What has changed
Set column to be not null

# How to test?
Test locally 

# Links
https://trello.com/c/DUgevj8y

### CODE REVIEW

**Comment by LeoKHoward** [comment]
[CE case missing ceActualResponses](https://trello.com/c/DUgevj8y/1494-ce-case-missing-ceactualresponses)

File: patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE casev2.cases ALTER COLUMN ce_actual_responses SET NOT NULL;
```
Comment (NickGrantONS) [inline patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql:1]: you need to patch the data before you alter the table, on the line before. Patch it like this:

`UPDATE casev2.cases SET ce_actual_responses = 0 WHERE ce_actual_responses IS NULL;`

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
Needs a data patch

File: patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE casev2.cases SET ce_actual_responses = 0 WHERE ce_actual_responses IS NULL;
```
Comment (NickGrantONS) [inline patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql:1]: what happened to the `alter table` part?

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
The `alter table` part of this went missing... or maybe it was never there... I can't remember but I'm sure it was there

File: patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql (hunk 2 above)
Comment (NickGrantONS) [reply patches/main/3000_alter_ce_actual_responses_to_be_not_null.sql:1]: `ALTER TABLE casev2.cases ALTER COLUMN ce_actual_responses SET NOT NULL;` <--- why was this removed?

**Review by NickGrantONS — APPROVED** [review]
Looks good. ATs passing.

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**